web/src/lib/components/chat/DropdownGroup.svelte

```diff
@@ -8,8 +8,29 @@
   import { languageStore } from '$lib/store/language-store';
   import { strategies, selectedStrategy, fetchStrategies } from '$lib/store/strategy-store';
   import { patternVariables } from '$lib/store/pattern-store';
+  import { currentSession, setSession } from '$lib/store/chat-store';
+  import { sessionAPI, sessions } from '$lib/store/session-store';
   import { onMount } from 'svelte';
 
+  let sessionInput = '';
+  let sessionsList: string[] = [];
+
+  // Update sessions list when sessions store changes
```

Comment:
Reactive statement is not necessary here. Could be simplified like this:

```javascript
$: sessionsList = $sessions?.map(s => s.Name) ?? [];
```

This is more concise and handles the null case explicitly.